db/clickhouse_migrate/20231024084411_create_events_raw.rb

```diff
@@ -2,7 +2,7 @@ class CreateEventsRaw < ActiveRecord::Migration[7.0]
   def change
     options = <<-SQL
       MergeTree
-      ORDER BY (organization_id, external_subscription_id, code, timestamp)
+      ORDER BY (organization_id, external_subscription_id, code, transaction_id, timestamp)
```

Comment:
I'm surprised to see transaction_id here. I expect it would be UUID or this kind of random string. Does it help with the deduplication?

Reply:
@julienbourdeau  Exactly, Clickhouse uses the order by as the primary key.
Adding the transaction_id here ensure that a transaction_id cannot appears more than once in a query result